### PR TITLE
Send signup confirmation email

### PR DESCRIPTION
**Pull request by martinzugnoni**
Solves https://github.com/rmotr/rmotr-sis/issues/42

### CODE REVIEW

File: rmotr_sis/accounts/views.py [hunk 1]
```diff
@@ -17,6 +18,10 @@ def form_valid(self, form):
         user.set_password(form.cleaned_data['password'])
         user.save()
 
-        # TODO: Send confirmation email
+        # send confirmation email
+        subject = 'Thank you for signing up at rmotr.com'
+        from_email = 'no-reply@rmotr.com'
+        send_template_mail(subject, 'signup-successful.html',
```
Comment (santiagobasulto) [inline rmotr_sis/accounts/views.py]: I'd make `from_email` an optional parameter to send_template_email that takes 'no-reply@rmotr.com' by default. 99% of the emails will have that same from_email.

File: rmotr_sis/rmotr_sis/settings/base.py [hunk 2]
```diff
@@ -81,3 +81,6 @@
 AUTH_USER_MODEL = 'accounts.User'
 
 CRISPY_TEMPLATE_PACK = 'bootstrap3'
+
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+EMAIL_TEMPLATE_LOCATION = os.path.join(BASE_DIR, 'templates', 'emails')
```
Comment (santiagobasulto) [inline rmotr_sis/rmotr_sis/settings/base.py]: `EMAIL_TEMPLATE_LOCATION = os.path.join(TEMPLATE_DIRS, 'emails')`

**Comment by santiagobasulto** [comment]
2 small comments. +1

File: rmotr_sis/rmotr_sis/settings/base.py (hunk 2 above)
Comment (martinzugnoni) [inline rmotr_sis/rmotr_sis/settings/base.py]: `TEMPLATE_DIRS ` is a tuple.

Comment (santiagobasulto) [inline rmotr_sis/rmotr_sis/settings/base.py]: `EMAIL_TEMPLATE_LOCATION = os.path.join(TEMPLATE_DIRS[0], 'emails')`

If we move templates to somewhere else, EMAIL_TEMPLATE_LOCATION shouldn't change.

Comment (martinzugnoni) [inline rmotr_sis/rmotr_sis/settings/base.py]: +1

**Pull request merged**